libqtile/xcbq.py

```diff
@@ -316,20 +317,20 @@ def alloc_color(self, color):
         """
             Flexible color allocation.
         """
-        if color.startswith("#"):
-            if len(color) != 7:
+        try:
+            return self.conn.conn.core.AllocNamedColor(
+                self.cid, len(color), color
+            ).reply()
+        except xcffib.xproto.NameError:
+            if len(color) < 6:
                 raise ValueError("Invalid color: %s" % color)
```

Comment:
I see your point.
I think I will wait for tych0 and hope he is ok with it .
I have spent a lot of unnecessary time on this little Bug :stuck_out_tongue: